recipes/unix.rb

```diff
@@ -71,6 +73,7 @@
   owner node['android-sdk']['owner']
   group node['android-sdk']['group']
   backup node['android-sdk']['backup_archive']
+  strip_components node['android-sdk']['legacy_sdk'] ? 1 : 0
```

Comment:
Major fix for unix system is just simply extract archive to `android-sdk` directory without strip directory (by default ark enter to the first directory and then extract the content (default value: `1`))